traceapp/app.go

```diff
@@ -39,6 +39,8 @@ type App struct {
 
 	tmplLock sync.Mutex
 	tmpls    map[string]*htmpl.Template
+
+	log bool
```

Comment:
good catch, def. agree on letting lib users decide to disable app logging, fixed on: https://github.com/chris-ramon/appdash/commit/66b5c2e05e5debe6690b9af214b98c76ade806fe